util/loadutil/cpu.go

```diff
@@ -15,12 +15,30 @@
 package loadutil
 
 import (
+	"fmt"
+	"github.com/cubefs/cubefs/util/log"
 	"time"
 
 	"github.com/shirou/gopsutil/cpu"
 )
 
-func GetCpuUtilPercent(sampleDuration time.Duration) float64 {
-	utils, _ := cpu.Percent(sampleDuration, false)
-	return utils[0]
+func GetCpuUtilPercent(sampleDuration time.Duration) (used float64, err error) {
+	utils, err := cpu.Percent(sampleDuration, false)
+	if err != nil {
+		log.LogErrorf("[GetCpuUtilPercent] err: %v", err.Error())
+		return
+	}
+	if utils == nil {
```

Comment:
This part is unnecessary.

If `utils` is nil, `len(utils)` will return `0`.

Reply:
See also: [doc](https://pkg.go.dev/builtin#len).

Reply:
it's for this case: if the utils ni not nil but its len is 0

cpu.Percent( ) may be modified in the future， this check may prevent possible exceptions

Reply:
> it's for this case: if the utils ni not nil but its len is 0
> 
> cpu.Percent( ) may be modified in the future， this check may prevent possible exceptions

We may just need `if len(utils) == 0`?